install/installer/pkg/common/common.go

```diff
@@ -431,7 +431,7 @@ func ConfigcatEnv(ctx *RenderContext) []corev1.EnvVar {
 		},
 		{
 			Name:  "CONFIGCAT_BASE_URL",
-			Value: "https://" + ctx.Config.Domain + "/configcat",
+			Value: "http://proxy.default.svc.cluster.local:9547/configcat",
```

Comment:
```suggestion
			Value: "http://" + ClusterAddress(proxy.Component, ctx.Namespace, port) 
```

Reply:
I will do but after testing, don't want to wait for another build.